cloud/aws/templates/aws_oidc/app.tf

```diff
@@ -211,7 +211,7 @@ module "civiform_metrics_scraper_container_def" {
   container_memory_reservation = 1024
 
```

Comment:
**thought:** Trying to think of why this wouldn't work. Should we attempt to only build this scraper container if the metrics are enabled? Similar to what we did with the count for other resources?

Reply:
I tried that and terraform doesn't allow the number of containers in a task def to change based on a variable

Reply:
Hmm would it work if you updated this container_definitions variable based on that variable? https://github.com/civiform/cloud-deploy-infra/blob/40c339725e080607a22708279e320ce656ff3cab/cloud/aws/templates/aws_oidc/app.tf#L353

Reply:
That's what I tried updating, Terraform doesn't want that to change. I guess an alternative could be to create multiple task definitions and switch which one we use based on the variable?